pyproject.toml

```diff
@@ -7,6 +7,7 @@ dependencies = [
     "pytest>=8.3.5",
     "httpx>=0.28.1",
     "coverage>=7.6.1",
+    "datamanager"
```

Comment:
This might actually be the right location.

Reply:
yea, this treats it more properly like a workspace